addons/website/models/website.py

```diff
@@ -470,10 +470,14 @@ def set_images(images):
         tour_asset_id = self.env.ref('website.configurator_tour')
         tour_asset_id.copy({'key': tour_asset_id.key, 'website_id': website.id, 'active': True})
 
-        # logo was generated as base64 url
-        logo = kwargs.get('logo')
-        if logo:
-            website.logo = logo.split(',', 1)[1]
+        # Set logo from generated attachment or from company's logo
+        logo_attachment_id = kwargs.get('logo_attachment_id')
+        company = website.company_id
+        if logo_attachment_id:
+            attachment = self.env['ir.attachment'].browse(logo_attachment_id)
+            attachment.res_field = 'logo'
```

Comment:
Still strange, why setting this here but res_id and res_model at first creation on the JS side? :thinking: Can't it all be here?